YOLayout/YOLayout_IMP.m

```diff
@@ -195,25 +195,7 @@ - (CGRect)setFrame:(CGRect)frame view:(id)view {
   return frame;
 }
 
-#pragma mark Common Layouts
-
-+ (YOLayout *)vertical:(NSArray *)subviews margin:(UIEdgeInsets)margin padding:(CGFloat)padding {
-  return [self layoutWithLayoutBlock:[YOLayout verticalLayout:subviews margin:margin padding:padding]];
-}
-
-+ (YOLayoutBlock)verticalLayout:(NSArray *)subviews margin:(UIEdgeInsets)margin padding:(CGFloat)padding {
```

Comment:
I like the box much more than the predefined blocks. I've tried to use these a few times but ended up reverting to the more verbose layout code because I need to tweak something.